split up ecsql reference into indiviual files, remove iframed iModel … (#6377)
iTwin/itwinjs-core: docs/learning/ECSqlReference/ORDERBY.md

# ORDER BY clause

> Sort result by set of expressions in ascending or descending order. It is also use to order nulls in result set by putting them in front or last of results.

Reviewer: ```suggestion
Sort result by set of expressions in ascending or descending order. It is also used to order nulls in result set by putting them at the start or end of results.
```
